scripts/release/install_ffmpeg.sh

```diff
@@ -57,16 +57,40 @@ install_build_deps() {
   then
     echo -e "\x1b[32mInstalling dependent components and libraries via yum...\x1b[0m"
     ${SUDO} yum install pkg-config make gcc gcc-c++ nasm yasm freetype-devel -y
+    if [ "$ENABLE_SRT" = "true" ]; then
```

Comment:
Seem we do not need it.

Reply:
This is not only used by streaming-agent